enhancements/installer/aws-outposts-remote-workers.md

```diff
@@ -0,0 +1,215 @@
+---
+title: aws-outposts-remote-workers
+authors:
+  - "@nirarg"
+  - "@pkliczewski"
+reviewers:
+  - "@sdodson"
+  - "@JoelSpeed"
+  - "@Miciah"
+  - "@alebedev87"
+  - "@patrickdillon"
+  - "@jsafrane"
+  - "@damdo"
+approvers:
+  - "@sdodson"
+api-approvers:
+  - None
+creation-date: 2022-10-11
+last-updated: 2022-10-11
+tracking-link:
+  - https://issues.redhat.com/browse/OCPPLAN-9617
+  - https://issues.redhat.com/browse/ECOPROJECT-866
+see-also: []
+replaces: []
+superseded-by: []
+---
+
+
+# Remote workers on AWS outpost
+
+## Summary
+
+Running workload closer to end users or leveraging data locality seems to be
+important for many customers. That is why AWS came up with fully [managed
+service](https://docs.aws.amazon.com/outposts/latest/userguide/how-outposts-works.html)
+that extends AWS infrastructure and run on customer's premises. This enhancement
+proposes extending how we deploy OCP on AWS platform so it is possible to run on
+AWS Outpost.
+
+The work is divided into several phases:
+- Phase 0: Research. Understand what is needed to deploy OCP on AWS Outpost.
+  No code changes needed and we only provide [documentation](https://github.com/openshift/openshift-docs/pull/53265) on how to customize
+  generated manifests and list known limitations.
+- Phase 1: Improve cluster deployment on AWS Outposts by implementing necessary
+  changes and provide mitigations to know limitations.
+- Phase 2: Add OCP on AWS Outpost support to ROSA for regular clusters and HyperShift.
+
+## Motivation
+
+Users might want to use OCP on AWS Outpost running in their own data center.
+AWS Outposts enables customers to build and run applications on premises using
+the same programming interfaces as in AWS Regions, while using local compute
+and storage resources for lower latency and local data processing needs.
+
+### User Stories
+
+Phase 0:
+- As a cluster admin I want to follow OCP [documentation](https://github.com/openshift/openshift-docs/pull/53265) to deploy OCP on AWS Outpost
+  where my master nodes are running in the regular AWS region and workers in AWS Outpost.
+
+Phase 1:
+- As a cluster admin I want to use an OCP installer to deploy OCP on AWS Outpost.
+  (Cluster deployment topology depends on mitigating lack of support for NLB)
```

Comment:
Based on the load balancer limitations, is the expectation in phase 1 that only compute nodes--not control plane are deployed to outposts?

Reply:
Yes, correct. Only worker nodes are deployed to the outposts.